### PR TITLE
feat(auth): change connexion panel layout

### LINKED ISSUE #1683: feat(auth): change connexion panel layout
- [x] Swap "Se connecter" et "Créer un compte" 
- [x] remove "étudiant" / "partenaire"

- [ ] First form is : Sign In 
  - Email / password 
  
- [x] Than the button "Créer un compte" to `/signup/student`

### PR DESCRIPTION
**Pull request by JNSPK**
<p align =center>${\color{orange}FEAT}$ </p></p><p align=center></p><p align =right>fix #1683 </p>

### CODE REVIEW

**Comment by douglasduteil** [comment]
The "or" section is not following the design

### Expected : 
image

### Actual : 
image

**Review by douglasduteil — APPROVED** [review]

File: apps/www/app/(home)/LoginForm.tsx [hunk 1]
```diff
@@ -15,12 +15,12 @@ export function LoginForm({
...
   return (
-    <div className="grid grid-cols-1 gap-5 rounded-md border bg-white px-4 py-5 text-Dove_Gray shadow-[10px_10px_10px_#00000029]">
+    <div className="grid grid-cols-1 gap-5 rounded-md border bg-white px-4 py-5 text-[#BEBEBE] shadow-[10px_10px_10px_#00000029]">
       <Login onSubmit={onLogin} />
       <div className="inline-flex w-full items-center justify-center px-3">
-        <hr className="my-4 h-px w-full border-0 bg-Dove_Gray " />
-        <span className="absolute left-1/2 -translate-x-1/2 bg-white px-3 font-medium uppercase text-Dove_Gray ">
-          ou
+        <hr className="my-4 h-px w-full border-0 bg-[#BEBEBE] " />
+        <span className="absolute left-1/2 -translate-x-1/2 bg-white px-3 text-[#BEBEBE] ">
```
Comment (douglasduteil) [inline apps/www/app/(home)/LoginForm.tsx:22]: question: The text is already BEBEBE from the parent div ?

Comment (JNSPK) [reply apps/www/app/(home)/LoginForm.tsx]: yep. i removed unnecessary css

**Review by douglasduteil — APPROVED** [review]
👌

**Pull request merged**